tasks/add_new_prebid_partner.py

```diff
@@ -324,22 +394,24 @@ def main():
       {name_start_format}Owner{format_end}: {value_start_format}{user_email}{format_end}
 
     Line items will have targeting:
-      {name_start_format}hb_pb{format_end} = {value_start_format}{prices_summary}{format_end}
+      {name_start_format}{hb_pb_key}{format_end} = {value_start_format}{prices_summary}{format_end}
       {name_start_format}hb_bidder{format_end} = {value_start_format}{bidder_code}{format_end}
-      {name_start_format}placements{format_end} = {value_start_format}{placements}{format_end}
+      {name_start_format}placements{format_end} = {value_start_format}{placements}{format_end}{additional_keys}
 
     """.format(
       num_line_items = len(prices),
       order_name=order_name,
       advertiser=advertiser_name,
       user_email=user_email,
+      hb_pb_key=hb_pb_key,
       prices_summary=prices_summary,
       bidder_code=bidder_code,
       placements=placements,
       sizes=sizes,
       name_start_format=color.BOLD,
       format_end=color.END,
       value_start_format=color.BLUE,
+      additional_keys=additional_keys
     ))
 
   ok = input('Is this correct? (y/n)\n')
```

Comment:
Could you add a couple tests to `test_add_new_prebid_partner.py` to test the validations of the new settings?